.gitignore

```diff
@@ -8,6 +8,5 @@
 /*.log
 /.vscode
 /.idea
-package-lock.json
```

Comment:
This is to prevent someone accidentally adding `package-lock.json` to a PR. Is there a reason to remove it?

Reply:
One of the Fixtures had a package-lock.json, tarvis tests were failing because this wasn't getting committed to Github

Reply:
OK so now you have committed it, you should be able to revert this change and everything still passes. Alternatively you can change it to `/package-lock.json` and it should only affect the one in root.

Reply:
I guess changing it to `/package.json` would be better, this is to make sure no one accidentally deletes the required `package-lock.json` in the  `_fixtures`